scripts/hook.py

```diff
@@ -440,20 +455,66 @@ def hacked_basic_transformer_inner_forward(self, x, context=None):
             x = self.ff(self.norm3(x)) + x
             return x
 
+        def hacked_group_norm_forward(self, *args, **kwargs):
```

Comment:
The function name "hacked_group_norm_forward" seems to suggest hacking the group_norm. But it actually seems to be hacking the forward functions of the "ResBlock" and "SpatialTransformer" blocks. 
Am I correct in understanding that it hacks the forward functions of the blocks, rather than the group norm?

Reply:
The intention is to hack any features before feed it to group norm - but it seems not important as we can apply adain to any layers.

Reply:
I understand, thank you!